app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected_result", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
```

Comment:
The test cases for (15, 15, [1, 1]) and (23, 23, [1, 1]) appear to be testing the same scenario. Ensure that your tests don't repeat the same case multiple times as per the checklist.